model/project_ref.go

```diff
@@ -2287,8 +2285,13 @@ func SaveProjectPageForSection(projectId string, p *ProjectRef, section ProjectP
 				},
 			})
 	case ProjectPageGithubAppSettingsSection:
-		// TODO: Implement in DEVPROD-5995.
-		return false, nil
+		err = db.Update(coll,
+			bson.M{ProjectRefIdKey: projectId},
+			bson.M{
+				"$set": bson.M{
+					projectRefGithubPermissionGroupByRequester: p.GitHubPermissionGroupByRequester,
```

Comment:
Can we rename this to projectRefGithubPermissionGroupByRequesterKey to be consistent with key naming? (also same for `projectRefGitHubDynamicTokenPermissionGroups`)